.github/workflows/health_base.yaml

```diff
@@ -0,0 +1,155 @@
+# A CI configuration to check PR health.
+
+name: Health:Base
+
+# Callers of this workflow should use it as follows:
+#
+# name: Health
+# on:
+#   pull_request:
+#     branches: [ main ]
+#     types: [opened, synchronize, reopened, labeled, unlabeled]
+# jobs:
+#   health:
+#     uses: dart-lang/ecosystem/.github/workflows/health.yaml@main
+#     with:
+#       coverage_web: true #If the coverage should run browser tests
+
+# Callers may optionally specify the version of the SDK to use when running the
+# health check. This can be useful if your package has a very recent minimum SDK
+# constraint. This is done via the `sdk` input parameter. Note that this
+# parameter is not required; it defaults to `stable` - using the most recent
+# stable release of the Dart SDK.
+# 
+# The checks can also be restricted to any subset of version, changelog, and license,
+# if needed.
+#
+# jobs:
```

Comment:
Nit: I find it more readable to have only one snippet of documentation with all possible options and a comment per option.